### PR TITLE
Space/enter for "Guess" and "Next Round"

### PR DESCRIPTION
**Pull request by EliteAsian123**
## Types of Changes

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Description
While in-game, you can now press `space` or `enter` to guess a selected location. You can also use `space` or `enter` to proceed to the next round while in the round summary screen.

### CODE REVIEW

**Review by BilelJegham — COMMENTED** [review]
Hello, thanks for your contribution.

Don't you think it will be great to not only focus on the next button?

It would be great to think "Keyboard accessibility" of all parts of GeoGuess : 
 - Game definition: settings, mode, maps. Add accessibility to the card settings 
 - Guessing, define the location with your keyboard
 
 Best regards

File: src/components/Maps.vue [hunk 1]
```diff
@@ -107,12 +107,7 @@
                     ($viewport.width > 450 || isMakeGuessButtonClicked)
             "
             id="guess-button"
-            :disabled="
-                randomLatLng == null ||
-                    selectedPos == null ||
-                    isGuessButtonClicked ||
-                    (!!this.room && !isReady)
-            "
+            :disabled="!canGuess()"
```
[the remark this answers is not in the document]
Comment (EliteAsian123) [reply src/components/Maps.vue:110]: Yes, sorry. Again.

**Comment by EliteAsian123** [comment]
Yes, if I have time in the future I will look into adding more keyboard accessibility to GeoGuess. Thanks!

**Comment by EliteAsian123** [comment]
Hello, congrats on v1.0.0. I've confirmed that this pull request still works on the new version.

**Comment by BilelJegham** [comment]
Hello @EliteAsian123, 
As said below, this PR is not enough to improve the user experience. We need to think globally by adding "Keyboard accessibility" across the platform.

Not only on one button.
It will be complete if we can :
 1. start a game
 2. play
   a. Select position
   b. Guess
   c. Go to the next round

**Review by BilelJegham — CHANGES_REQUESTED** [review]
Ok for the code.

But it's only on the submit btn

**Comment by EliteAsian123** [comment]
Hi. Unfortunately, I do not have the time to add Global Keyboard Accessibility at the moment so I will be closing the pull request. If I decide to take it on in the future, I will re-open the pull request. Thanks!

**Pull request closed without merging**